apps/sms/views/shared/style/panel.css

```diff
@@ -5,34 +5,51 @@
   top: 0;
   bottom: 0;
 
-  transform: translate(100%);
-
   overflow: auto;
 
   background: white;
 
   animation: .2s both linear;
 }
+html[dir="ltr"] .panel { transform: translate(100%); }
+html[dir="rtl"] .panel { transform: translate(-100%); }
 
-.panel-active {
+/**
+ * adding html[dir] to make the specificity of this rule high enough to
+ * overwrite the default translation for element with class panel, without
```

Comment:
maybe you can add "just above" so that it's easier to understand you're talking about the rules that are just above.